tools/agents/model-watch.py

```
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+"""מנטר שהמודלים שבקונפיג עדיין קיימים אצל הספקים.
+
+**למה.** ביום החיבור הראשון זה קרה פעמיים: ‏groq/compound החליף שם,
+ו-`llama-3.1-8b` של Cloudflare הוצא משימוש חודשים קודם — ואיש לא
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)